versions.md

```diff
@@ -16,6 +16,9 @@ Forthcoming
     * Several methods that were meant for internal use have been removed from public API
     * All public methods now have `snake_case` names
     * All public attributes now have `snake_case` names
+  * RTPSParticipant:
```

Comment:
We may also mention the new `get_*_builtin_topic_data()` in the DataWriter/Reader API